diagrams/utils.py

```diff
@@ -0,0 +1,30 @@
+import contextvars
```

Comment:
I don't prefer the `utils` as filename personally. I think it is better to locate these functions in `__init__` file.

Reply:
I tried to remove the `utils` as for your request but that causes circular imports issue.
I think it is a small thing that shouldn't prevent merging this PR